scripts/create-database.js

```diff
@@ -1,56 +1,54 @@
-const { Client } = require('pg')
-const path = require('path')
+const { Client } = require("pg");
+const path = require("path");
 
 // capture first command line argument passed to this script
-const envName = process.argv.slice(2)[0]
+const envName = process.argv.slice(2)[0];
 
 // this function decides whether to load .env or .env.test.
 
 const loadEnv = (envName) => {
-  const { NODE_ENV } = process.env
-  if (NODE_ENV != 'production') {
+  const { NODE_ENV } = process.env;
+  if (NODE_ENV != "production") {
+    const envFile = envName === "test" ? "../.env.test" : "../.env";
 
-    const envFile = envName === 'test' ? '../.env.test' : '../.env'
-
-    require('dotenv').config({
+    require("dotenv").config({
       path: path.join(__dirname, envFile),
-    })
+    });
 
     // capture the name of the database so we can create it
-    const databaseName = process.env.PGDATABASE
+    const databaseName = process.env.PGDATABASE;
 
     // remove the name of the database from the environment, so pg doesn't try to connect to a db which doesn't exist yet
-    delete process.env.PGDATABASE
+    delete process.env.PGDATABASE;
 
-    return databaseName
+    return databaseName;
   }
-}
+};
 
 const createDatabase = async (databaseName) => {
   // create a new client, it will automatically load the connection details from process.env
-  const client = new Client()
+  const client = new Client();
   try {
-    await client.connect()
+    await client.connect();
 
-    console.log(`Creating ${databaseName} database...`)
+    console.log(`Creating ${databaseName} database...`);
 
-    await client.query(`CREATE DATABASE ${databaseName}`)
+    await client.query(`CREATE DATABASE ${databaseName}`);
 
-    console.log('Database created!')
+    console.log("Database created!");
   } catch (err) {
-
     switch (err.code) {
       // this is the postgres error code for when a database already exists. You could store this in a constant to make the code more readable
       case "42P04":
-        console.log('Database already exists!')
-        break
+        console.log("Database already exists!");
```

Comment:
I know the track says its play to copy this in to the codebase, but it doesn't mean you should leave in questions like this which are only there to help you think about possible optimisations. Think of it from the perspective of a possible new employer, things like this (and the commented out title at the top of the file) make it especially clear that this was pasted in from somewhere which doesn't leave a great impression

I would always say its much more beneficial to you while learning these things to get into the idea of typing your work out as it helps you get accustomed to the flow of work, its more engaging mentally, and you get the same code in the end. To be clear, this isnt a criticism since we said its okay to copy paste in this instance, just a suggestion from someone whos being doing this for years and has noticed the difference in doing that :)